include/slate/BaseMatrix.hh

```diff
@@ -483,10 +483,15 @@ public:
     void tileRelease( int64_t i, int64_t j, int device=HostNum );
 
     //--------------------------------------------------------------------------
-    template <Target target = Target::Host>
     void tileSend(int64_t i, int64_t j, int dst_rank, int tag = 0);
 
-    template <Target target = Target::Host>
+    template <Target target>
+    [[deprecated( "Setting the Target of tileSend is deprecated. Will be removed 2024-12." )]]
+    void tileSend(int64_t i, int64_t j, int dst_rank, int tag = 0)
+    {
+        tileSend( i, j, dst_rank, tag );
+    }
```

Comment:
Looks like `tileSend` never used the target, right? And target doesn't make sense for send.